### ISSUE TITLE
Review the Ziggurat Report and check if there are any missing network fixes

### ISSUE THREAD

**Issue by teor2345** [ZcashFoundation/zebra#3278]
## Motivation

We need to check if we missed anything from the Ziggurat report, or if there is anything we have a different opinion about now.

### Scheduling

We should try to fix any important issues from the report first.

We have already removed some of the issues from the report from the Zebra release candidate. We don't need to fix those issues before reviewing the report.

**Comment by teor2345** [comment]
Let's do this later in January, when we've fixed the Ziggurat issues that are in the release candidate epic.

**Comment by mpguerra** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Engineering-Team-5fb24d9264a3e8000e666a9e/issues/ZcashFoundation/zebra/3278?planning-poker) with ZenHub @conradoplg @dconnolly @jvff @oxarbitrage @teor2345 @upbqdn

**Comment by mpguerra** [comment]
I assume that the outcome of this issue will be a list of new issues to deal with the missing network fixes? Either that or we decide that there are no fixes to be made? :D

**Comment by teor2345** [comment]
Yes, I'll put any proposed issues in:
- #3413

**Comment by mpguerra** [comment]
Increasing priority to High, if we're going to be adding any more issues to the Stable Release Epic we need to make a decision on this sooner rather than later.

**Comment by teor2345** [comment]
Here is how I reviewed the report:

1. Go to the list of test statuses at https://github.com/eqlabs/ziggurat#test-status
2. Find tests where `zcashd` passed, but Zebra failed (if they both failed, I assumed the test itself has a bug)
3. If we haven't fixed the failures already, open a ticket

We also reviewed the two PDF reports that the Ziggurat team sent us privately.

I opened the following tickets:
- [Use optimised Zebra with tested dependency versions for Ziggurat Tests](https://github.com/eqlabs/ziggurat/pull/109) (they were using a debug build of Zebra for their tests)
- #3429
